models/packages/package.go

```diff
@@ -43,6 +43,7 @@ const (
 	TypeNuGet     Type = "nuget"
 	TypePub       Type = "pub"
 	TypePyPI      Type = "pypi"
+	TypeRpm       Type = "rpm"
```

Comment:
Maybe RPM

Reply:
Lowercase is fine here, things like NPM could also be written uppercase, but weren't.

Reply:
I used `TypeRpm` because the library I use to extract the package file also uses `Rpm`. If we want to name it `TypeRPM` it's ok too.

Reply:
I like `Rpm`.